python/tox.ini

```
@@ -0,0 +1,59 @@
+[testenv]
+deps = .[test]
+commands =
+    py39: python -m unittest
+allowlist_externals =
+    behave
+
+[flake8]
+extend-select = W504
+ignore = E501, E402, D202, D401, W503, D212
+exclude = .git,__pycache__,docs/source/conf.py,old,build,tests,dist,venv,.env,.venv,.eggs,.tox
+max-complexity = 10
+# The GitHub editor is 127 chars wide
+max-line-length = 127
+statistics = True
+show-source = True
+docstring-convention = google
+count = True
+per-file-ignores =
+    # Ignore docstring errors in tests
+    *tests/*:D100,D101,D102,D103,D104,D417,D200,ANN101,ANN201,W293,W291,ANN001,E731,W605
+    *src/aac/plugins/**/templates/**:D104
+
+[unittest]
+start-dir = .
+code-directories = src
+plugins = nose2.plugins.mp
+
+[multiprocess]
+always-on = True
+processes = 4
+
+[coverage]
+coverage = src
+always-on = True
+coverage-config = tox.ini
+
+[run]
+branch = True
+omit =
+  *.jinja2
+
+[report]
+fail_under = 80.00

```

```suggestion
fail_under = 90.00

```

We should be trying to maintain the 90% coverage till we find out we are unable to for some reason.